internal/resources/server/resource.go

```diff
@@ -38,52 +37,79 @@ type Resource struct {
 // input configuration and returns a slice of server network data which can be
 // used in a POST request to the PCBe API to create a new server
 // TODO: (API) Issue FF-31496 will prevent this from working currently.
-func parseNetworksToPostFormat(serverNetworks basetypes.ListValue) (
+func parseNetworksToPostFormat(dataP *ServerModel) (
 	[]privatecloudbusiness.
 		V1beta1SystemsItemAddHypervisorServersPostRequestBody_serverNetworkable,
 	error,
 ) {
 	var postRequestNetworks []privatecloudbusiness.
 		V1beta1SystemsItemAddHypervisorServersPostRequestBody_serverNetworkable
 
+	serverNetworks := (*dataP).ServerNetwork
+	if serverNetworks.IsNull() {
+		msg := "server network list is null"
```

Comment:
For making these more compact, any reason you're not just doing e.g. `errors.New("server network list is null")` ? And below.

Reply:
I've done this in other parts of the code where the message length is longer. I typically have a preference to keep line lengths short (<80). It also means you have the option to log the (same) message string if you want.

You're right that in this case it's not adding much, only some consistency with existing code.